feat(changelog): rabbitmq-added-cloud-essentials-for-rabbitmq-moves-to-g 2026-08-04 (#6751)
scaleway/docs-content: changelog/august2026/2026-08-04-rabbitmq-added-cloud-essentials-for-rabbitmq-moves-to-g.mdx

---

> title: Cloud Essentials for RabbitMQ moves to General Availability!

Reviewer: ```suggestion
title: Cloud Essentials for RabbitMQ moves to General Availability
```

status: added
date: 2026-08-04
category: integration-services
product: rabbitmq
---

> Cloud Essentials for RabbitMQ moves to General Availability!

Reviewer: ```suggestion
[Cloud Essentials for RabbitMQ](/rabbitmq/) is now in General Availability.
```
